snow/uptime/manager.go

```diff
@@ -68,33 +69,38 @@ func (m *manager) StartTracking(nodeIDs []ids.NodeID, subnetID ids.ID) error {
 
 		durationOffline := now.Sub(lastUpdated)
 		newUpDuration := upDuration + durationOffline
-		if err := m.state.SetUptime(nodeID, subnetID, newUpDuration, now); err != nil {
+		if err := m.state.SetUptime(nodeID, newUpDuration, now); err != nil {
 			return err
 		}
 	}
-	m.trackedSubnets.Add(subnetID)
+	m.startedTracking = true
 	return nil
 }
 
-func (m *manager) StopTracking(nodeIDs []ids.NodeID, subnetID ids.ID) error {
+func (m *manager) StopTracking(nodeIDs []ids.NodeID) error {
+	// TODO: this was not here before, should we add it?
+	if !m.startedTracking {
+		return nil
+	}
+	defer func() {
+		m.startedTracking = false
```

Comment:
it seems that `startedTracking` replaces `trackedSubnets` but only for the primary network.
but I don't see any code in the previous revision for `m.trackesSubnets.Remove`

Reply:
I think this was an issue previously. we typically use this function after shutdown, so we never restart tracking. but I think the code itself should support restarting in any case.